packages/application-shell/src/components/redirect-to-project-create/redirect-to-project-create.spec.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { renderApp } from '../../test-utils';
```

Comment:
Using `renderApp` as we need the router to be setup for the dev mode `LinkButton`.